src/processor/reconnection-graph.service.ts

```diff
@@ -516,7 +504,8 @@ export class ReconnectionGraphService {
       if (e instanceof Error && e.message.includes('Inability to pay some fees')) {
         // in case capacity is low pause the queue
         this.graphUpdateQueue.pause();
-        throw e;
+        this.logger.error("Capacity is low: job processing queue paused.");
+        return;
```

Comment:
instead of throwing an error and queue already failing job, queue NT update and pause the queue